Docs/details/events.md

```diff
@@ -0,0 +1,154 @@
+# Events
+
+Events are a notion core to how Santa interacts with a sync-server. Events are generated when there is a blocked `exec()` while in lockdown or monitor mode. Events are also generated in monitor mode for an `exec()` that was allowed to run, but would have been blocked in lockdown mode. This allows an admin to roll out Santa to their macOS fleet in monitor mode, but still collect meaningful data. The events collected while in monitor can be used to build a fairly comprehensive whitelist of signing certificates and binaries before switching the fleet to lockdown mode.
```

Comment:
I'd probably change exec() to execve() here as all other exec*() functions call up to execve. It also catches posix_spawn, which is separate.